application/common/models/PasswordHistory.php

```diff
@@ -58,6 +58,11 @@ private function hasAlreadyBeenUsedWithinLimit(): bool
         return false;
     }
 
+    public function expires(): string
+    {
+        return MySqlDateTime::format(strtotime('+1 year', strtotime($this->created_utc)));
```

Comment:
Can we move the `+1 year` to a config parameter and then pull from env with a default of `+1 year`? Different IdPs may want different policies.